arch/risc-v/src/esp32c3-legacy/esp32c3_idle.c

```diff
@@ -200,21 +200,13 @@ static void up_idlepm(void)
             esp32c3_pmsleep(CONFIG_PM_SLEEP_WAKEUP_SEC * 1000000 +
                                 CONFIG_PM_SLEEP_WAKEUP_NSEC / 1000);
           }
-          break;
```

Comment:
this is a kind of minor, but what is the purpose of removing the `break` here? Behaviour seems to be the same with and without fallthru

Reply:
This modification has no impact, it just removes redundant code.